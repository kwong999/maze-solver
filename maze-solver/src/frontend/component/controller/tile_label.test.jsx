import React from 'react';
import renderer from 'react-test-renderer';
import TileLabel from './tile_label';

test('TileLabel should render without error', () => {
  const component = renderer.create(
    <TileLabel />
  )
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
})