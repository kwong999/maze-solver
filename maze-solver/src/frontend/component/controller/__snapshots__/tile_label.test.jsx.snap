// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TileLabel should render without error 1`] = `
<div
  className="maze-action"
>
  <label>
    <p>
      Label:
    </p>
    <dl>
      <dt
        className="solution"
      />
      <dd>
        : Solution path
      </dd>
      <dt
        className="used"
      />
      <dd>
        : Searched Tile
      </dd>
    </dl>
  </label>
</div>
`;
